File: test11/devowel_file.pl
#!/usr/bin/perl
$file = $ARGV[0];
open FILE,'<', $file;
@f= <FILE>;
close FILE;
foreach $line (@f){
	$line =~ s/[AEIOUaeiou]*//g;
}
open $ff, '>' ,$file;
foreach(@f){
	print $ff "$_";
}
close $ff;

File: test11/devowel_file.py
#!/usr/bin/python3
import sys,re,os
file = sys.argv[1]
f = open(file,"r")
w = open("copy","w")
for line in f:
	line = re.sub("[AEIOUaeiou]*","",line)
	w.write(line)
#	f.write(line)
w.close
f.close
os.rename("copy", file)
